app/components/public/speaker-item.js

```diff
@@ -5,5 +5,13 @@ export default Component.extend({
   classNames  : ['four wide  speaker column'],
   socialLinks : computed(function() {
     return this.get('speaker').getProperties('twitter', 'facebook', 'github', 'linkedin');
+  }),
+  hasSocialLinks: computed(function() {
```

Comment:
there is no need for this. `socialLinks` returns an array, if it's size is 0, you can just use `{{if socialLinks}}` to check if there is any social link.

Reply:
@CosmicCoder96 The `socialLinks` will always be returning a dictionary of 
`{twitter: <null or value>, facebook: <null or value>, github: <null or value> linkedin: <null or value>}` hence the size will always be 4, also, it doesn't handle the case where the user might have some biographical / short description.